### PR TITLE
Fix

### PR DESCRIPTION
**Pull request by Raphaeljunior**
Parenthesis Fix

### CODE REVIEW

**Comment by Raphaeljunior** [comment]
@jridgewell done

File: test/test-errortracker.js [hunk 1]
```diff
@@ -432,7 +432,7 @@ global code@https://abc.cdn.ampproject.org/v/s/abc/doc?amp_js_v=0.1`;
     at mf.zc (https://cdn.ampproject.org/rtv/031496877433269/v0.js:408:166)
     at pf (https://cdn.ampproject.org/rtv/031496877433269/v0.js:112:409)
     at lf.$d (https://cdn.ampproject.org/rtv/031496877433269/v0.js:115:86)
-    at https://cdn.ampproject.org/rtv/031496877433269/v0.js:114:188`;
+    at (https://cdn.ampproject.org/rtv/031496877433269/v0.js:114:188)`;
```
Comment (jridgewell) [inline test/test-errortracker.js]: This is incorrect. We only use parenthesis if there's a function name.

File: test/test-errortracker.js [hunk 2]
```diff
@@ -11,7 +11,7 @@
  * distributed under the License is distributed on an "AS-IS" BASIS,
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions andc
- * limitations under the License.
+ * limitations under the License. Travis check
```
Comment (jridgewell) [inline test/test-errortracker.js]: Remove.

File: test/test-errortracker.js (hunk 1 above)
Comment (jridgewell) [inline test/test-errortracker.js]: Still incorrect.

Comment (jridgewell) [inline test/test-errortracker.js]: Still incorrect.

File: test/test-errortracker.js [hunk 3]
```diff
@@ -364,21 +364,21 @@ describe('Test stacktrace conversions are done correctly', function() {
     at mf.zc (https://cdn.ampproject.org/rtv/031496877433269/v0.js:408:166)
     at pf (https://cdn.ampproject.org/rtv/031496877433269/v0.js:112:409)
     at lf.$d (https://cdn.ampproject.org/rtv/031496877433269/v0.js:115:86)
-    at https://cdn.ampproject.org/rtv/031496877433269/v0.js:114:188`;
+    at (https://cdn.ampproject.org/rtv/031496877433269/v0.js:114:188)`;
```
Comment (jridgewell) [inline test/test-errortracker.js]: Incorrect.

**Pull request merged**